packages/@aws-cdk/core/lib/fs/options.ts

```diff
@@ -85,9 +75,30 @@ export interface CopyOptions {
 }
 
 /**
- * Options related to calculating source hash.
+ * Obtains applied when copying directories into the staging location.
```

Comment:
```suggestion
 * Options applied when copying directories
```

(this is a general utility used by asset staging but can technically be used for other purposes)